src/components/form/Form.vue

```diff
@@ -161,6 +167,24 @@ export default {
                     if(input.type == 'captcha') {
                         this.containsCaptcha = true;
                     }
+                    if(input.type == 'label') {
+                        columns.push({
+                            columnWidth: input.columnWidth || 12,
+                            content: {
+                                component: 'text',
+                                data: input || {type: 'label', text: "label"},
```

Comment:
Why are you adding the `type` here again inside the data of the component

Reply:
my bad , I didn't realize that inputTypes already add them , will remove them @AbdAsh